### DISCUSSION TITLE
How to disable SPA when  page transition?

### DISCUSSION THREAD

**Discussion by mogi86** [nuxt/nuxt#16921] [Questions]
Hi. Nuxt3 uses universal rendering. In this rendering mode, my understanding is that  application run as `spa` when access via `NuxtLink`. Is my understanding correct? So, do I need to use `<a>` tag if I don’t want to use `spa` when page transition?

**Comment by TheAlexLichter** [comment]
Why would you not want SPA-like navigation in the first place?

**Reply by mogi86** [reply]
I know SPA is good for performance and user experience. However, I don't want the content to load asynchronously on some pages.
This may not necessary, but if what I want to achieve is possible I'd like to know the way.
